packages/grafana-data/src/transformations/index.ts

```diff
@@ -3,18 +3,6 @@ export * from './transformers/ids';
 export * from './matchers';
 export { standardTransformers } from './transformers';
 export * from './fieldReducer';
-export { FilterFieldsByNameTransformerOptions } from './transformers/filterByName';
-export { FilterFramesByRefIdTransformerOptions } from './transformers/filterByRefId';
-export { SeriesToColumnsOptions } from './transformers/seriesToColumns';
-export { ReduceTransformerOptions } from './transformers/reduce';
-export { LabelsToFieldsOptions } from './transformers/labelsToFields';
-export {
-  CalculateFieldTransformerOptions,
-  CalculateFieldMode,
-  getResultFieldNameForCalculateFieldTransformerOptions,
-} from './transformers/calculateField';
-export { OrganizeFieldsTransformerOptions } from './transformers/organize';
-export { createOrderFieldsComparer } from './transformers/order';
```

Comment:
No need to export these anymore!  they can be essentially internal

Reply:
Yeah, but this leads to `grafana-data/src...` imports. Maybe we could export this via as a namespace, similar as we do with the actual transformers (`standardTransformers`) and mark it @internal so that it's not included in the docs at least?

Reply:
BUT... IIUC, 

```
@grafana-data/src
@grafana-ui/src
```
is OK in the core  (it is used very often!) but bad bad bad in the packages since it breaks plugins.

We obviously need some rules/linting to help enforce this!

Reply:
Yes it's OK in the core:) just thinking loud here :)